test/test_sparse_csr.py

```diff
@@ -396,6 +394,34 @@ def test_csr_matvec(self, device, dtype):
             with self.assertRaisesRegex(RuntimeError, "mv: expected"):
                 csr.matmul(bad_vec)
 
+    @onlyCUDA
```

Comment:
Test this for CPU as well?

Reply:
Yes, This PR was waiting the merge of PR https://github.com/pytorch/pytorch/pull/58768